packages/gatsby/index.d.ts

```diff
@@ -1162,10 +1162,15 @@ export interface Actions {
 
   /** @see https://www.gatsbyjs.com/docs/reference/config-files/actions/#unstable_createNodeManifest */
   unstable_createNodeManifest(
-    this: void, 
-    args: { manifestId: string, node: Node }, 
+    this: void,
+    args: {
+      manifestId: string
+      node: Node
+      updatedAt?: string | number
+      daysSinceLastUpdate?: number
```

Comment:
We'll want to write a test for this new functionality as well. There are integration tests for createNodeManifest in `integration-tests/node-manifest`

Reply:
Just saw this and remembered there are unit tests as well! We should add a unit test and an integration test 👍 
`/packages/gatsby/src/utils/__tests__/node-manifest.ts`